priv/templates/script.upload.eex

```diff
@@ -125,4 +149,14 @@ if [ "$(uname -s)" = "Darwin" ]; then
     fi
 fi
 
-cat $FILENAME | ssh -s $SSH_OPTIONS $DESTINATION nerves_firmware_ssh2
+# Set fd 3 to screen so fwup progress still shows
+exec 3>&1 
+
+if ! err=$(cat $FILENAME | ssh -s $SSH_OPTIONS $DESTINATION nerves_firmware_ssh 2>&1 >&3) && [[ "$err" =~ "subsystem request failed" ]];
+then
+    # Old subsystem on port 8989 might still be used, so try that
+    printf "fwup:$FILESIZE,reboot\n" | cat - $FILENAME | ssh -s -p 8989 $SSH_OPTIONS $DESTINATION nerves_firmware_ssh
+else
+    # Something else broke. Output it
+    echo -e $err
+fi
```

Comment:
So I'm 100% positive I have an ending newline here. Shows locally. I'm thinking GitHub just doesn't show it in the file?